ruleset/sca/amazon/cis_amazon_linux_2.yml

```diff
@@ -398,7 +398,7 @@ checks:
       - pci_dss: ["2.2.4"]
       - nist_800_53: ["CM.1"]
       - tsc: ["CC5.2"]
-    condition: all
+    condition: none
```

Comment:
If we set the condition to `none` it means the rule has to match the opposite you want to achieve. In this case the rule should be:
```
- 'c:systemctl show autofs.service -> r:unitfilestate=enabled'
```
In this case, the rule will be positive when Autofs is enabled, since the condition is `none` the check will fail as expected.

Generally, when negating twice in the same check (condition and negate rule), means that the logic can be simplified.